app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -22,26 +23,40 @@ import {
 } from '@oxide/api'
 import { Storage24Icon } from '@oxide/design-system/icons/react'
 
+import { HL } from '~/components/HL'
 import { DiskStateBadge } from '~/components/StateBadge'
 import { AttachDiskSideModalForm } from '~/forms/disk-attach'
 import { CreateDiskSideModalForm } from '~/forms/disk-create'
 import { getInstanceSelector, useInstanceSelector } from '~/hooks/use-params'
+import { confirmAction } from '~/stores/confirm-action'
 import { addToast } from '~/stores/toast'
 import { useColsWithActions, type MenuAction } from '~/table/columns/action-col'
 import { Columns } from '~/table/columns/common'
 import { Table } from '~/table/Table'
 import { Button } from '~/ui/lib/Button'
+import { CreateButton } from '~/ui/lib/CreateButton'
 import { EmptyMessage } from '~/ui/lib/EmptyMessage'
-import { TableEmptyBox } from '~/ui/lib/Table'
+import { TableControls, TableEmptyBox, TableTitle } from '~/ui/lib/Table'
 
 import { fancifyStates } from './common'
 
-const EmptyState = () => (
+const BootDiskEmptyState = () => (
   <TableEmptyBox>
     <EmptyMessage
       icon={<Storage24Icon />}
-      title="No disks"
-      body="Attach a disk to this instance to see it here"
+      title="No boot disk set"
+      // TODO: boot order docs link
+      body="Read about boot order LINK HERE"
```

Comment:
Just highlighting that this isn't set yet

Reply:
<img width="1222" alt="Screenshot 2024-10-01 at 2 23 55 PM" src="https://github.com/user-attachments/assets/9e29836f-d469-46a7-b37b-634b00485ac8">

We don't really tell them with this copy how to set the boot disk, or that it's done via the table below. I'm having trouble coming up with an elegant way to say that, though. (Copy might also need to handle the "no attached disks at all" case with the copy, so that could be even more ungainly.)

Reply:
One more thought, not really pinned to this code, but pertinent to the screenshot above: The two states are `attached` and `detached`; is it redundant to have the `STATE` column here? There's no way that a disk could show up here in a `detached` state, right? Or might we add some third state in time?

(I also get it if it looks a little empty without that column.)

Reply:
I've also noticed that. It is redundant but it also looks empty without it. I think it's ok because it reinforces the concept of attachment.